src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -150,7 +150,7 @@ describe('sidebar.components.sidebar-content', function() {
     });
   }
 
-  beforeEach(
+  const makeSidebarContentController = () => {
```

Comment:
There might be a better way to do this and I feel like what I did was kinda hacky but I didn't want to have to refactor all the tests so I abstracted this to a function so that I could call it to re-build the controller after I initialized the things needed to calculate directLinkedGroupFetchFailed as the digest wasn't doing it for me.

Reply:
> that only gets initialized when the controller is constructed, so it is expected that it can only change in one direction afterwards

Yes, exactly. I can add a comment about that so it's clearer-should have done that earlier.